python/tvm/relay/frontend/onnx.py

```diff
@@ -836,6 +837,192 @@ def _impl_v1(cls, inputs, attr, params):
         return Gelu._impl_v1([inp], attr, params)
 
 
+class EmbedLayerNormalization(OnnxOpConverter):
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        input_ids = inputs[0]
+        segment_ids = inputs[1]
+        word_emb = inputs[2]
+        pos_emb = inputs[3]
+        segment_emb = inputs[4]
+        gamma = inputs[5]
+        beta = inputs[6]
+
+        mask = inputs[7]
+        pos_ids = inputs[8]
+
+        eps = attr["epsilon"] if "epsilon" in attr else 1e-12
```

Comment:
where'd you get these default values? 

Also suggest attr.get('epsilon', 1e-12)

Reply:
`1e-12` is commonly used for layer norm epsilon, but I couldn't quite find the default value in onnxrt (please let me know if you find it). We could also enforce the presence of epsilon, but onnxrt doesn't do this.

Reply:
https://github.com/microsoft/onnxruntime/commit/6e341c002f6760baa5b08c6970d9cd994730a835#diff-534f8d4a645c4fa2e80983b37196e601f7a69424e93bbcb4008a75d18427b8d6R30 

Appears to be 1e-12